### PR TITLE
MWPW-163025: CCD Merch cards UI observations

### PR DESCRIPTION
**Pull request by Axelcureno**
Fixes UI observations of CCD cards.

Resolves: [MWPW-163025](https://jira.corp.adobe.com/browse/MWPW-163025)

**Test URLs:**
- Before: https://main--milo--adobecom.aem.page/libs/features/mas/docs/ccd.html?theme=dark
- After: https://mwpw-163025--milo--adobecom.aem.page/libs/features/mas/docs/ccd.html?theme=dark

### CODE REVIEW

File: libs/features/spectrum-web-components/src/theme.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import '@spectrum-web-components/theme/sp-theme.js';
 import '@spectrum-web-components/theme/theme-light.js';
+import '@spectrum-web-components/theme/theme-darkest.js';
```
Comment (yesil) [inline libs/features/spectrum-web-components/src/theme.js:3]: @Axelcureno please do not update this one which will impact catalog page performance.
src/themes/darkest.js one below should be sufficient.

File: libs/features/mas/src/variants/ccd-suggested.css.js [hunk 2]
```diff
@@ -6,58 +6,77 @@ export const CSS = `
...
 }
 
 merch-card[variant="ccd-suggested"] [slot="heading-xs"] {
-  color: var(--spectrum-gray-800, #F8F8F8);
   font-size: var(--merch-card-heading-xxs-font-size);
   line-height: var(--merch-card-heading-xxs-line-height);
 }
 
+sp-theme[color='light'] merch-card[variant="ccd-suggested"] [slot="heading-xs"] {
+  color: #222;
```
Comment (yesil) [inline libs/features/mas/src/variants/ccd-suggested.css.js:34]: why spectrum colors cannot be used here?

File: libs/features/mas/src/variants/ccd-suggested.css.js [hunk 3]
```diff
@@ -6,58 +6,77 @@ export const CSS = `
...
 }
 
-sp-theme[color='light'] merch-card[variant="ccd-suggested"] [slot="body-xs"] {
-  color: var(--ccd-gray-700-dark, #464646);
-}
-
-sp-theme[color='dark'] merch-card[variant="ccd-suggested"] [slot="body-xs"] {
-  color: var(--ccd-gray-100-light, #F8F8F8);
+sp-theme[color='darkest'] merch-card[variant="ccd-suggested"] {
+  background-color: #1D1D1D;
```
Comment (yesil) [inline libs/features/mas/src/variants/ccd-suggested.css.js:15]: Can we only use spectrum variables for colors, fonts, etc?

**Review by yesil — COMMENTED** [review]
I have concerns with hard coded colors. 
If there are spectrum variables providing those values, we should be using them.

File: libs/features/spectrum-web-components/src/theme.js (hunk 1 above)
Comment (Axelcureno) [reply libs/features/spectrum-web-components/src/theme.js]: Removed 👍🏼

**Review by yesil — APPROVED** [review]

File: libs/features/mas/docs/ccd.html [hunk 4]
```diff
@@ -44,7 +44,7 @@
       const params = new URLSearchParams(document.location.search);
       const darkTheme = params?.get('theme')?.toLowerCase() === 'dark';
       const theme = document.createElement('script');
-      theme.setAttribute('src', `../../spectrum-web-components/dist/themes/${darkTheme ? 'dark' : 'light'}.js`);
+      theme.setAttribute('src', `../../spectrum-web-components/dist/themes/${darkTheme ? 'darkest' : 'light'}.js`);
```
Comment (3ch023) [inline libs/features/mas/docs/ccd.html:47]: Nit: if we change from 'dark' to 'darkets' we should rename all occurances of 'dark' to 'darkest', e.g. 'darkTheme' and line 66:
`<a href="?theme=dark">Dark</a>`

**Comment by 3ch023** [comment]
Hi @Axelcureno is this PR also includes changes for https://jira.corp.adobe.com/browse/MWPW-162957?
could you split it in 2 PRs please?
one for fixing width in https://jira.corp.adobe.com/browse/MWPW-163025 and another for https://jira.corp.adobe.com/browse/MWPW-162957 (I am not sure we should use 'darkest', CCD confirmed that they can use 'dark')

**Comment by Axelcureno** [comment]
Closing this PR since it's no longer needed (ticket items addressed in other PRs)

**Pull request closed without merging**